Update mdo-email-entity-page.md (#8349)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/mdo-email-entity-page.md

- *Exchange transport rules (also known as mail flow rules or ETRs)*: These rules are applied to a message at the transport layer and take precedence over phish and spam verdicts. Mail flow rules are created and modified in the Exchange admin center at <https://admin.exchange.microsoft.com/#/transportrules>, but if any mail flow rule applies to a message, the rule name and GUID will be shown here. Valuable information for tracking purposes.

> - *Primary Override: Source*: Primary override and source refers to the tenant or user setting which impacted the delivery of the email, overriding the delivery location given by system (as per the threat and detection technology). As an example, this could be an email blocked due to a tenant configured transport rule or an email allowed due to an end user setting for Safe Senders. 

Reviewer: ```suggestion
- *Primary Override: Source*: Primary override and source refer to the tenant or user setting which impacted the delivery of the email, overriding the delivery location given by the system (as per the threat and detection technology). As an example, this could be an email blocked due to a tenant configured transport rule or an email allowed due to an end-user setting for Safe Senders. 
```

> - *All Overrides*: All Overrides refer to the list of overrides (tenant or user setting) which got applied on the email, and which may or may not have impacted the delivery of an email. As an example, if a tenant configured transport rule as well as tenant configured policy setting (e.g. from Tenant Allow Block list) is applied to an email, then both will be listed in this field. You can check the primary override field to determine the setting which impacted the delivery of the email. 

Reviewer: ```suggestion
- *All Overrides*: All Overrides refer to the list of overrides (tenant or user settings) that was applied on the email, which may or may not have impacted the delivery of an email. As an example, if a tenant configured transport rule, as well as a tenant configured policy setting (for example, from the Tenant Allow Block list), is applied to an email, then both will be listed in this field. You can check the primary override field to determine the setting that impacted the delivery of the email. 
```
